.github/workflows/win.yml

```diff
@@ -0,0 +1,50 @@
+name: Test
+
+on:
+  push:
+    paths-ignore:
+    - "docs/**"
+  pull_request:
+    paths-ignore:
+    - "docs/**"
+
+jobs:
+  test:
+
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [windows-latest]
+        requirements: [latest]
+        python-version: [3.8]
+    name: ${{ matrix.os }}, Python ${{ matrix.python-version }}, ${{ matrix.requirements }}
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Cache pip
+      uses: actions/cache@v1
+      with:
+        path: ~/.cache/pip
+        key: ${{ runner.os }}-pip-${{ matrix.python-version }}-${{ hashFiles('**/requirements*.txt') }}
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Update pip
+      run: pip install --upgrade pip wheel
+    - name: Install pip dependencies
+      run: pip install -r requirements/dev.txt
+    - name: Compile all the files
+      run: python -m compileall -q -f .
+    - name: Setup build
+      run: python setup.py --quiet build
```

Comment:
When doing the cleanup here, those are probably not needed in none of the actions. The setup.py build is tested in separate workflow and it should byte compile the files as well (I guess the reasoning behind this was syntax check, but we do have lint job as well).

```suggestion
```

Reply:
Interesting, the build looks to be failing after removing this. goin to try and add it back to see if it works again

Reply:
Adding it back fixed it, but here is the failure when both these steps were removed https://github.com/translate/translate/runs/610261179?check_suite_focus=true#step:8:13 if that rings a bell why

Reply:
And failures when only Setup step is removed https://github.com/translate/translate/pull/3981/checks?check_run_id=610316715#step:11:17

Reply:
That's certainly bug in the code, but I don't get why byte-compiling hides it. I've tried to address this in https://github.com/translate/translate/pull/3984

Reply:
OK, after that lands I can try and remove the steps again

Reply:
I've just merged that.

Reply:
Looks like it's catching something else now https://github.com/translate/translate/pull/3981/checks?check_run_id=610959605#step:10:28

Reply:
OK, add a suggested fix from pylint. Hope it makes sense